nni/tools/nnictl/nnictl_utils.py

```diff
@@ -222,9 +222,18 @@ def stop_experiment(args):
     if experiment_id_list:
         for experiment_id in experiment_id_list:
             print_normal('Stopping experiment %s' % experiment_id)
-            experiments_config = Experiments()
-            experiments_dict = experiments_config.get_all_experiments()
-            rest_pid = experiments_dict.get(experiment_id).get('pid')
+            # retry up to 10 times to get the experiment metadata
```

Comment:
Only stop has this problem?

Reply:
No, a better way is to put this retry into `Experiments`, but in this way, we need to align both `python` and `ts` code.

Reply:
Well, in that case, add a FIXME in code as reminder.

Reply:
add it